yt_dlp/extractor/whyp.py

```diff
@@ -0,0 +1,45 @@
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class WhypIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?whyp\.it/tracks/(?P<number_id>[0-9_-]+)/(?P<display_id>[a-zA-Z0-9_-]+)'
```

Comment:
Are you sure that the `display_id` is unique?

It seems like sometimes it can just be the title of the track (slug-ified), e.g. this URL: https://www.whyp.it/tracks/76863/vacation

Reply:
I have changed the id to the number_id. The number_id appears to be unique and the url resolves to the correct audio page without the display_id.